lib/utils/socketChainHelper.js

```diff
@@ -127,6 +128,22 @@ function getAssertErrorMessage(res, text, data) {
 	return `${res.errorType}: "${text}"${errors}`;
 }
 
+/**
+ * Create human readable general error message from suitest error response
+ * @param {Object} res
+ * @returns {string}
+ */
+function getGeneralErrorMessage(res) {
+	const errorType = path(['result', 'errorType'], res);
+	const errorReason = path(['result', 'message', 'info', 'reason'], res) || {};
+
```

Comment:
@andriichumak , we changed reason type of IL (errorHandler ret.errorType) for video adapter errors only from string to object with code and message. I think it would be better if we keep it as string and add another key for message `errorText` etc, if no changes from BE are required.

Reply:
Yeah, I changed that in IL in that way specifically to avoid extra work on BE. I'll discuss that with Tomas.